src/core/thread.d

```diff
@@ -4854,16 +4858,21 @@ private:
                 finalHandler = reg.handler;
             }
 
-            pstack -= EXCEPTION_REGISTRATION.sizeof;
+            // With safeseh(?), the chain must not extend to the very top
```

Comment:
> Shouldn't the change also go into the upstream implementation, though? (We can just cherry-pick the fix from master afterwards.)

I don't think dmd will ever support safeseh. It shouldn't do harm to add it, though.